rp2040-hal/src/pll.rs

```diff
@@ -161,24 +150,26 @@ impl<D: PhaseLockedLoopDevice> PhaseLockedLoop<Disabled, D> {
             return Err(Error::PostDivOutOfRage);
         }
 
-        let ref_freq_range: Range<Hertz<u32>> = Hertz(5_000_000)..vco_freq.div(16);
+        use fugit::RateExtU32;
+        let ref_freq_max_vco = (vco_freq.to_Hz() / 16).MHz();
```

Comment:
this should be `let ref_freq_max_vco = (vco_freq.to_Hz() / 16).Hz();` Note the dropped M from the last call

Since you convert vco_freq to Hz the value, even divided by 16 will be to big for HertzU32.Mhz() to handle